src/request/sign-transaction/SignTransaction.js

```diff
@@ -148,7 +153,9 @@ class SignTransaction {
 
         const publicKey = key.derivePublicKey(request.keyPath);
         const signature = key.sign(request.keyPath, request.transaction.serializeContent());
-        const result = /** @type {SignTransactionResult} */ {
+
+        /** @type {KeyguardRequest.SignTransactionResult} */
```

Comment:
And that cast?

Reply:
That cast is not _necessary_ anymore, but it helps in identifying a missing property (because the error is then on the object, and not on the `resolve()` call.

Reply:
Ok